include/efx/THibaFire.h

```diff
@@ -5,6 +5,11 @@
 
 namespace efx {
 struct THibaFire : public TForever4 {
```

Comment:
use enums for particle id here, please

Reply:
have revamped particle ID system of repo c: